### PR TITLE
Update credits.

### PR DESCRIPTION
**Pull request by NotHexadecimal**
The credits, they are updated now.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-- Java: [@Nylon](https://github.com/nylone)
-- C: [@Nylon](https://github.com/nylone)
+Hi, this is just another repo with random brainfuck compilers in it.
+
+Implementations:
+- Javascript: [@NotHexadecimal](https://github.com/NotHexadecimal)
+- Go: [@TheTipo01](https://github.com/TheTipo01)
+- Rust: [@techmccat](https://github.com/techmccat)
+- Lua: [@Nylon](https://github.com/nylone)
+- Mips: [@Nylon](https://github.com/nylone)
```
Comment (techmccat) [inline readme.md:8]: Linking the same profile every time is aviodable boilerplate
Proposed fix:
- Lua: [nilo](https://github.com/nylone)
- Mips: also nylo
- Java: also also neelon
- C: also also also nylone

Comment (NotHexadecimal) [reply readme.md:8]: Fair enough, fix incoming.

File: readme.md [hunk 2]
```diff
@@ -1,13 +1,18 @@
 Hi, this is just another repo with random brainfuck compilers in it.
 
 Implementations:
```
Comment (techmccat) [inline readme.md:3]: Remove brackets around handles

**Review by techmccat — APPROVED** [review]
lgtm

**Pull request merged**